### ISSUE TITLE
UDPipe-1.2.0

### ISSUE THREAD

**Issue by ALeczkowski** [ufal/udpipe#146]
Hello, is there a way to change random seed (to randomize it instead of heaving it fixed) from command line when running training with UDPipe?

**Comment by foxik** [comment]
No, that is unfortunately currently not possible.

As a hack, you could try permuting the order of sentences on input, that might change the IDS of words and order of sentences during training, which might have a similar effect.

**Comment by ALeczkowski** [comment]
Thanks for your answer!

I was hoping for an easy and simple solution. Since there is not one, I will modify the source code myself and compile UDPipe. However, I have an issue with compilation. I trained the same model with precompiled UDPipe and with UDPipe compiled by me (of course I did not implement any modifications to the source code yet); however, I get a bit different results. Any idea what might be the issue? Can it be due to this error during compilation:

`warning: dynamic exception specifications are deprecated in C++11`

If you have any obvious solutions that I am missing I would appreciate if you share them with me. 

PS. I did install all of the required programs (using Ubuntu).
